js/widgets/forms/select.js

```diff
@@ -25,119 +25,110 @@
 			"../../widget",
 			"../../zoom",
 			"../../navigation/path",
-			"./reset" ], factory );
+			"../widget.theme",
+			"jquery-ui/form-reset-mixin" ], factory );
 	} else {
 
 		// Browser globals
 		factory( jQuery );
 	}
 } )( function( $ ) {
 
-return $.widget( "mobile.selectmenu", $.extend( {
+return $.widget( "mobile.selectmenu", $.widget( "mobile.selectmenu", [ {
```

Comment:
this call is weird i think it would be more clear to separate out adding the theme